ogzaf/demographics.py

```diff
@@ -346,13 +515,12 @@ def pop_rebin(curr_pop_dist, totpers_new):
     return curr_pop_new
 
 
-def get_imm_resid(totpers, min_yr, max_yr):
+def get_imm_resid(totpers, min_yr, max_yr, graph=False):
```

Comment:
Why not use the net-migration figures from the UN (indicator 65)?

Reply:
@SeaCelo. I would leave the `graph=False` parameter and default here, just because there is a graphing option. Regarding your question, as we discussed last week, the key is that the population numbers must be consistent. We can either take the population, fertility, and mortality data and back out the immigration rates (as we do here), or we can take the fertility, mortality, and immigration data and use that to get the population data. A third option is to take the population, fertility, mortality, and immigration rate data from the UN database and just hope they are consistent with each other. This is not likely true.

This topic is a little bit nuanced and will take a deeper discussion. But the approach currently taken in `demographics.py` is valid.

Reply:
@rickecon Agreed. Though I am convinced the UN Population data is consistent with itself (they also take a residual approach), using the current system works well, so lets keep it as you suggest.